src/App.jsx

```diff
@@ -9,6 +9,7 @@ import { TYPE_FORM } from './constants';
 import HomePage from './pages/HomePage/HomePage.jsx';
 import ProductsPage from './pages/ProductsPage/ProductsPage.jsx';
 import MainLayout from './components/MainLayout/MainLayout.jsx';
+import HomePage from './pages/HomePage/HomePage.jsx';
```

Comment:
what the hell is this. It crushing my build. How did it work for you?

Reply:
+ deleted